### ISSUE TITLE
1.12.2 [version 043.0] Vehicle Interaction - Client Crash 

### ISSUE THREAD

**Issue by Pdan4** [MrCrayfish/MrCrayfishVehicleMod#228]
Circumstance: fresh world, client logged in, moved around a bit, and crashed.

```
java.lang.ClassCastException: java.lang.Integer cannot be cast to net.minecraft.nbt.NBTTagCompound
    at com.mrcrayfish.vehicle.common.CommonEvents.onPlayerInteract(CommonEvents.java:305)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_2770_CommonEvents_onPlayerInteract_RightClickBlock.invoke(.dynamic)
```

**Comment by MrCrayfish** [comment]
Check for new updates before reporting issue. This has been fixed in 0.43.5

**Comment by Pdan4** [comment]
My apologies. Thank you!
